### DISCUSSION TITLE
cpu and memory limits to install superset on Kubernetes

### DISCUSSION THREAD

**Discussion by prashanthvulapu** [apache/superset#25199] [Q&A / Help]
Hi Everyone, We are planning to do a small POC by installing superset on Kubernetes. Can you please help me out with different services available within Superset and what are the CPU and memory min & max limits for each of these services ? Note - The max data volume can be up to 1 GB. Thanks in advance.

**Comment by iilrgmv** [comment]
Hello! Any suggestions on this?? Same question here!
